src/SubmissionExporter.php

```diff
@@ -75,6 +75,7 @@ public function data(Submission $submission) {
       'action' => url("node/$nid", $link_options),
       'submission' => url("node/$nid/submission/{$submission->sid}", $link_options),
     ];
+    $data['_optins'] = OptInExporter::fromConfig()->export($submission);
```

Comment:
prefixing with `_` is a hint/convention that it's not a plain value?

Reply:
Yep. For historic reasons `action` is not prefixed, but I can’t change that now.

Actually I would push all form values into their own sub-array that would make things a lot easier. But then again I’d head for separate endpoints for each event type anyway. I don’t think there is an advantage to breaking things now instead of later.

Reply:
thanks for explaining!